src/style.css

```diff
@@ -1,3 +1,75 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  background-color: azure;
+  font-weight: 500;
+  font-style: normal;
+  height: 100%;
+}
+
+.header {
+  background-color: #fff;
+  display: flex;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin: 10px 0;
+  display: block;
+}
+
+.main-nav {
+  margin-left: auto;
+  box-sizing: border-box;
+}
+
+.nav-list {
+  padding: 0;
+  margin: 0;
+  text-align: center;
+  box-sizing: border-box;
+  display: flex;
+}
+
+.nav-list li {
```

Comment:
Don't use tag names for styling (except html and body) , better use classes.

![image](https://github.com/mate-academy/layout_moyo-header/assets/54860641/0b43619b-5a32-489e-9e5b-5f8436b705fa)